app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -30,6 +30,28 @@ def inherited_attributes
           inherited_attrs
         end
 
+        def apply_inherited_attributes(attributes, initialized = true)
+          attributes = super(attributes, initialized)
+          facet_attrs = attributes['content_facet_attributes']
+          return attributes if facet_attrs.blank?
+          cv_id = facet_attrs['content_view_id']
+          lce_id = facet_attrs['lifecycle_environment_id']
+          if initialized && (cv_id.blank? || lce_id.blank?)
```

Comment:
If both `cv_id` and `lce_id` are blank, what then?

I considered adding explicit logic for this. But I think we're okay without it, because the two cases are

1. Host with content facet: will already have a valid LCE and CV, and the host will use its existing values
2. Host without content facet: `facet_attrs` will be `nil`, meaning this code path isn't touched

anyway, I couldn't get it to error in my testing. But let me know if I'm wrong :D